src/js_native_api_v8.cc

```diff
@@ -1718,6 +1718,46 @@ napi_status NAPI_CDECL node_api_create_property_key_utf16(napi_env env,
   });
 }
 
+napi_status node_api_create_property_key_utf8(napi_env env,
```

Comment:
Please implement the `node_api_create_property_key_utf8` the same way as the `node_api_create_property_key_utf16` implemented using the `v8impl::NewString` helper function.
The current implementation is missing a few checks and uses `v8::NewStringType::kNormal` instead of `v8::NewStringType::kInternalized` which differentiates V8 normal string creation from a V8 string used as a property key.

Reply:
I tried to apply it as you suggested, thank you very much